deploy/katello.spec

```diff
@@ -116,8 +116,9 @@ Summary:    Common runtime components of %{name}
 Requires:       wget
 Requires:       curl
 Requires:       %{?scl_prefix}rubygem-katello
-Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli
 Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_foreman_docker
```

Comment:
Unfortunately, I think hammer-cli-katello now relies on hammer-cli-foreman-docker. Certainly the docker plugin should not be shipped w/ SAM, but the cli could be and then disabled by SAM install.

https://github.com/Katello/hammer-cli-katello/blob/master/hammer_cli_katello.gemspec#L42